Ll model setup

llong-unsw opened the pull request.

ProsperousPanda committed:
checkpoint

ProsperousPanda committed:
resolve merge conflict

Merge branch 'main' of https://github.com/llong-unsw/project-group-L into LL_model_setup

# Conflicts:
#	code/Data exploration.R

ProsperousPanda committed:
Merge branch 'main' of https://github.com/llong-unsw/project-group-L into LL_model_setup

ProsperousPanda committed:
modelling update

ProsperousPanda committed:
rename data exploration file

HaoChiou reviewed (approved):
I think it's pretty good

ProsperousPanda committed:
SVR checkpoint

ProsperousPanda committed:
finalised svr model

ProsperousPanda committed:
fix typos

ProsperousPanda committed:
finalise models

Merge branch 'main' of https://github.com/llong-unsw/project-group-L into LL_model_setup

# Conflicts:
#	code/02 Data exploration.R
